src/main/java/gift/option/model/Options.java

```diff
@@ -0,0 +1,28 @@
+package gift.option.model;
+
+import gift.common.exception.OptionException;
+import gift.option.OptionErrorCode;
+import java.util.List;
+
+public class Options {
+
+    private List<Option> options;
+
+    public Options(List<Option> options) {
+        this.options = options;
+    }
+
+    public List<Option> getOptions() {
+        return options;
+    }
+
+    public void validateDuplicated(Option option) throws OptionException {
+        options.forEach((it) -> it.validateDuplicated(option));
```

Comment:
단일 매개변수의 경우 괄호를 생략합니다
```suggestion
        options.forEach(it -> it.validateDuplicated(option));
```

Reply:
수정하였습니다!